tasks/modprobe.yml

```diff
@@ -1,3 +1,22 @@
 ---
 
-# modprobe
+- name: check if modprobe is already installed | os-10
+  command: 'modprobe -V'
+  register: modprobe_installed
+  ignore_errors: True
+  changed_when: False
+
+- name: install modprobe to disable filesystems | os-10
+  package:
+    name: 'kmod'
+    state: 'installed'
+  when: modprobe_installed and modprobe_installed.rc != 0
```

Comment:
I would change the state from `installed` to `present`. According to [the docs](https://docs.ansible.com/ansible/latest/package_module.html) `installed` should not work.